src/updatedb.py

```diff
@@ -166,10 +181,8 @@ def uploadResults(data, branch, revision, date):
                     elif (data2):
                         log = data2.get("blob", {}).get("logurl", "") 
 
-                    regression = 0
-
                     # Insert into MySQL Database
-                    sql = 'insert into testjobs (id, log, slave, result, duration, platform, buildtype, testtype, bugid, branch, revision, date, regression) values ('
+                    sql = 'insert into testjobs (id, log, slave, result, duration, platform, buildtype, testtype, bugid, branch, revision, date, failure_classification) values ('
```

Comment:
Please wrap the long line. Things seem to work fine if you use """ and format the query in a natural way.